packages/core/src/modules/oob/OutOfBandApi.ts

```diff
@@ -40,13 +43,20 @@ import { HandshakeReuseHandler } from './protocols/v1/handlers'
 import { HandshakeReuseAcceptedHandler } from './protocols/v1/handlers/HandshakeReuseAcceptedHandler'
 import { OutOfBandInvitation } from './protocols/v1/messages'
 import { V2OutOfBandService } from './protocols/v2/V2OutOfBandService'
-import { OutOfBandInvitation as V2OutOfBandInvitation } from './protocols/v2/messages'
+import { V2OutOfBandInvitation as V2OutOfBandInvitation } from './protocols/v2/messages'
 import { OutOfBandRecord } from './repository/OutOfBandRecord'
 
 const didCommProfiles = ['didcomm/aip1', 'didcomm/aip2;env=rfc19']
 
-export interface CreateOutOfBandInvitationConfig {
-  version?: 'v1' | 'v2'
+export enum OutOfBandVersion {
+  V11 = '1.1',
+  V20 = '2.0',
```

Comment:
```suggestion
  V1 = 'v1',
  V2 = 'v2',
```


Can we use the same syntax for this as the issue credential protocol? Not sure if it uses V2 or v2, but would like to keep this consistent across the framework